src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -128,13 +131,15 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
         obj.layers.set(layer.threejsLayer);
     };
     if (urlSuffix) {
-        const url = metadata.urlPrefix + urlSuffix;
+        let url = metadata.urlPrefix + urlSuffix;
+        if (urlSuffix.substring(0, 4) == 'http') {
```

Comment:
BTW, maybe you could rename `urlPrefix` -> `baseURL` and `urlSuffix` -> `url` or `path` ?